### ISSUE TITLE
"How RChain Changed My Life" (article contest)

### ISSUE THREAD

**Issue by allancto** [rchain/bounties#975]
### Benefit to RChain

1. Create buzz around RChain with personal stories
2. Create understanding of what RChain is like from personal perspectives 
3. Open to non-members
4. Invite new contributors

### Description 

For overall structure of article contests, see #974 (including contest rules, still in development). 

The article contest is open to anyone, members and non members alike. Because of the nature of this particular topic it may be well suited to existing RChain members, but a successful RChain will touch the lives of non members as well. 

Looking back at my personal experience, I was first drawn to RChain by the math and then by the Governance forum. Many others have expressed to me how riveting, inspiring, and so on the RChain experience has been. 

Please read this sample article created by @qelentium (submission unedited). Please consider submitting your own contribution. Submissions may be addressed for now to @owans (amachreeowanate@gmail.com) while we’re working on setting up proper mailboxes (contests@unofficialblog.rchain.world). Also the Colab channel #publications is a good place to get more information. 

### Budget and Objective

_Estimated Budget of Task:_ **$[2500-3500 prizes, 1000-3000 administration]**
_Estimated Timeline Required to Complete the Task:_ **[6 weeks, 1 preparation + 4 contest, 1 judging]**
_How will we measure completion?_ **[prizes awarded]**

### Legal

_Task Submitter shall not submit Tasks that will involve RHOC being transacted in any manner that (i) jeopardizes RHOC’s status as a software access token or other relevant and applicable description of the RHOC as an “asset”—not a security— or (2) violates, in any manner, applicable U.S. Securities laws

**Comment by casanwugo** [comment]
Awesome!!

**Comment by ysgjay** [comment]
This feels premature. I like the idea creating buzz, but how has RChain changed lives when it's not live yet?

**Comment by Viraculous** [comment]
@allancto this seems to me to a reinvention of the wheel, it should have been compressed with issue #974. I don't think the scope of this issue is different from that of #974.

**Comment by dckc** [comment]
Any feedback on whether this is on target?

 - [dreaming big at the RChain Governance Forum][1] | 2018-02-21  
_I'm back from the RChain Governance Forum in Seattle, which was full of big dreams, just like the first Web conference in Geneva ..._

[1]: http://www.madmode.com/2018/rchain-gov-forum-SEA.html

**Comment by allancto** [comment]
@owans would you say the "dreaming big" article is on target? @dckc I would say absolutely. 

As for having contest prizes for sharing experience, to the extent that recounting experiences such as yours is beneficial to RChain, helps others to understand what RChain is, what it can be, why RChain is worth engaging with, yes, this is an activity we should incentivize. Why would we limit incentivization to people who are already here working as employees or investors or directors or contractors? Anyone who has been touched by RChain and wants to write about it should be eligible, not just those of us (please pardon my french:) already at the trough. Contests are a good way to promote engagement and encourage contribution, while at the same time (unlike translation) having a defined and manageable budget. Written contributions also help to identify people with skills in articulation and organization, people we'd like to encourage to engage with our community.

**Comment by dckc** [comment]
> ... Why would we limit incentivization to people who are already here working as employees or investors or directors or contractors?

Because the impact of (perceived) astro-turfing could be disastrous. Witness the HashNode fiasco in #665

**Comment by dckc** [comment]
Oh... and this also looks like (a) a reasonably complete task proposal, not a discussion item, and (b) overlaps with the responsibilities of the Director of Content.

cf https://github.com/rchain/bounties/issues/974#issuecomment-424515275

**Comment by allancto** [comment]
@dckc I have reached out to Derek Beres, our Director of Content, but did not hear back yet (and I'm guessing he may want a couple of days to digest when he does have time to address our issue. 

I was never completely clear on the HashNode fiasco but it sounds like that was a case of something appearing to be an Official RNode statement or policy when it was not. That might lend credence to the idea that an Unofficial blog creates a complement to the Official blog (which as I've said I think is awesome and I also fully support). 

>Astroturfing is the practice of masking the sponsors of a message or organization (e.g., political, advertising, religious or public relations) to make it appear as though it originates from and is supported by grassroots participants.

If you're expressing concern about a conspiracy theory in which contests in our Unofficial Blog might be construed as "fake news" masking RChain as the sponsor of PR messages, I'd have to say that's not on my list of seriously negative possible events we should focus on avoiding in RChain's immediate future (such as lack of confidence in our governance systems). Contests are open, our submission process transparent. Articles are supposed to represent authentic experience. We can't truly avoid people saying they had great RChain experience they actually didn't, just because they want to win a contest, but I'd expect a discerning public to understand that.

**Comment by dckc** [comment]
We ran afoul of the HashNode community guidelines by having people paid to
post there.

So this is not a hypothetical.

**Comment by owans** [comment]
Yes it is! @allancto, reading through the article tells me how @dckc came to know about RChain, where it started, person's involved in the beginning, it's purpose and also noticed this content already on a blog,which is not difficult to retrieve since it's been written before now. 
Personally, this is my 3rd month around RChain, never being introduced to the world of Blockchain before, I was motivated by the fact that my skills would be encouraged and put to test, thanks to @allancto @jimscarver within a short time I can say "RChain has in a sense changed my life" as you can see in two publications to my name. 
Perhaps your use of the term 'astroturfing' could be ruled out if we would appreciate the different purpose of both blogs, also if people where to write genuine content such as yours @dckc. Also, I see no reason why I should write an article of up to 100 words if it wouldn't in one way or another be of benefit to me and to an extent others. The aim is to engage member participation and promote RChain, @dckc since you're up for people sharing their experiences about RChain, I encourage you be up for sponsoring too and possibly a judge in the contest.
@ysgjay I sincerely don't expect such comment from you,to an outsider such statement can be misconstrued.

**Comment by ddayan** [comment]
@owans See if marketing would want to sponsor it. Also it seems like you are really passionate about this is issue so you could offer to sponsor it as well.

**Comment by luigidemeo** [comment]
Developers in ethereum blog about ethereum to further their own careers and build up their own "thought leadership". They don't get paid just to pump out content. 

As discussed, this will need a sponsor. Paying people to share their experiences is not a scalable solution to community engagement.

**Comment by allancto** [comment]
@luigidemeo can you explain why defined budget contests are not scalable?

**Comment by luigidemeo** [comment]
@allancto paying people to share experiences is not scalable. The coop does not have an unlimited budget, nothing in the world that constitutes economic value is unlimited. The coop therefore must be prudent in how it allocates resources and must allocate to things that create the most relative value. Simply throwing money at attending meetings, writing up proposals, and having people writing about experiences means that we cannot allocate those funds to other needs such as building libraries to use rholang, or funding research....

**Comment by allancto** [comment]
I do see your statement that the coop must be prudent in how it allocates resources and must allocate to things that create the most relative value. I also see your opinion that meetings and proposals are worthless relative to the items you stated as being worth more. 

What i'm missing is an answer to my question, @luigidemeo can you explain why defined budget contests are not scalable?

**Comment by luigidemeo** [comment]
I said paying people to share experiences is not scalable. What that means is that if you pay people to share experiences, everybody who would have otherwise done it for free will now want to be paid. If everybody wants to be paid to share, we run into the issue of limited resources. 

From a macro perspective, a network cannot grow by paying for growth, it must grow from a higher purpose- money will only buy a certain amount of engagement and then when that money is gone, that engagement will fade as it was all a mirage @allancto

**Comment by allancto** [comment]
@luigidemeo this issue here is about a contest with a fixed number of prizes and a fixed prize budget. You mislead everyone who reads your (repeated) comments which misstate that this issue proposes to pay all. Your comments as a member of the community on issues such as this one are __important__. All of us want to consider your views thoroughly and to the best of our ability. You can help us all by paying careful attention to detail and choosing your words to be meaningful and correct in context. 

As far as higher purpose, macro perspectives and mirages, those again are colorful terms but again you're first overgeneralizing and then misleading. Certainly networks __can and often do__ sustainably "pay for growth" using advertising, discounts and many other means. Sustainability comes from the design of payment, not simply the fact. Your claim implicitly assigning yourself as the arbiter of the "macro perspective" is misleading. The higher purpose you refer to is what? Your metaphor of a mirage left by an unsuccessful Cooperative is evocative (yes, i like it, though in a clearly negative way!), but the subject of discussion here is _which_ path will lead to success and _which_ path will lead to mirage. You assert the opinion that anything other than single minded sponsorship of developers will lead to mirage, I assert the opposite. Toning down the drama will give us more focus on the actual issue. 

Thanks! -@allancto
